scripts/tests/run_tv_casting_test.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env -S python3 -B
+
+# Copyright (c) 2024 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import os
+import subprocess
+import sys
+import time
+
+import click
+
+# Configure logging format.
+logging.basicConfig(level=logging.DEBUG, format='%(levelname)s - %(message)s')
+
+LINUX_TV_APP_LOGS = './scripts/tests/Linux-tv-app-logs.txt'
+LINUX_TV_CASTING_APP_LOGS = './scripts/tests/Linux-tv-casting-app-logs.txt'
+
+RUN_INTERVAL = 2
```

Comment:
add unit of measurement to things ... this is `RUN_INTERVAL_SEC` ... but at the same time `run what?`.

It seems to be `TV_APP_STARTUP_TIME_SEC` or something like that. Please rename as appropriate to make the code easy to understand.

Reply:
Noted. I renamed the constant in the latest commit.